modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts

```diff
@@ -0,0 +1,558 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analytics-settings';
+import {switchChannel} from './utils/channel';
+import {
+	ACPage,
+	navigateTo,
+	navigateToACPageViaURL,
+	navigateToACWorkspace,
+} from './utils/navigation';
+import {addBreakdownByIndividualAttribute} from './utils/utils';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test(
+	'Empty state of no assets with data source and property.',
+
+	{
+		tag: '@LRAC-10405',
+	},
+
+	async ({apiHelpers, page}) => {
+		const randomString = getRandomString();
+
+		const siteName = 'My Site ' + randomString;
+
+		const site = await apiHelpers.headlessSite.createSite({
+			name: siteName,
+		});
+
+		const channelName = 'My Property' + randomString;
+
+		const {channel, project} = await syncAnalyticsCloud({
+			apiHelpers,
+			channelName,
+			page,
+			siteName,
+		});
+
+		await test.step('Go to Analytics Cloud and Switch the property', async () => {
+			await navigateToACWorkspace({page});
+			await switchChannel({
+				channelName,
+				page,
+			});
+		});
+
+		await test.step('Go to Assets page', async () => {
+			await navigateToACPageViaURL({
+				acPage: ACPage.assetPage,
+				channelID: channel.id,
+				page,
+				projectID: project.groupId,
+			});
+		});
+
+		await test.step('Go to Blogs, check empty state message', async () => {
+			await navigateTo({
+				page,
+				pageName: 'Blogs',
+			});
+
+			await expect(
+				page.getByText('There are no visitors data found.')
+			).toBeVisible();
+			await expect(
+				page.getByText(
+					'Check back later to verify if data has been received from your data sources, or you can try a different date range.'
+				)
+			).toBeVisible();
+			await expect(
```

Comment:
@rafaellajordao can you remove this statement, the below one should already check if the element is visible or not